src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const COST_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const LONG_PERIOD_DISCOUNT = 50;
+  const SHORT_PERIOD_DISCOUNT = 20;
+
+  return days >= LONG_TERM
+    ? days * COST_PER_DAY - LONG_PERIOD_DISCOUNT
+    : days >= SHORT_TERM
+      ? days * COST_PER_DAY - SHORT_PERIOD_DISCOUNT
+      : days * COST_PER_DAY;
```

Comment:
The nested ternary operator can be a bit hard to read. Consider refactoring to use if statements with return for better readability. This will also align with the checklist item 4 and 5, avoiding the 'else' after 'if' with 'return'.